bindings/python/tokenizers_manual_m1_build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+MACOSX_DEPLOYMENT_TARGET=10.11 SDKROOT="/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk" CFLAGS="-I/usr/include/openssl -I/usr/local/opt/readline/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include" CPPFLAGS="-I/usr/local/opt/zlib/include" LDFLAGS="-L/usr/lib -L/usr/local/opt/readline/lib" pyenv install <wtv_version>
+
+~/.pyenv/versions/<wtv_version>/bin/pip install setuptools wheel setuptools-rust==0.11.3 --ignore-installed --force-reinstall
+
+MACOSX_DEPLOYMENT_TARGET=10.11 ~/.pyenv/versions/<wtv_version>/bin/python setup.py bdist_wheel
+
```

Comment:
```suggestion
for VARIABLE in "3.7.12" "3.8.12" "3.9.10" "3.10.2"
do
    MACOSX_DEPLOYMENT_TARGET=10.11 SDKROOT="/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk" CFLAGS="-I/usr/include/openssl -I/usr/local/opt/readline/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include" CPPFLAGS="-I/usr/local/opt/zlib/include" LDFLAGS="-L/usr/lib -L/usr/local/opt/readline/lib" pyenv install $VARIABLE

    ~/.pyenv/versions/$VARIABLE/bin/pip install setuptools wheel setuptools-rust==0.11.3 --ignore-installed --force-reinstall

    MACOSX_DEPLOYMENT_TARGET=10.11 ~/.pyenv/versions/$VARIABLE/bin/python setup.py bdist_wheel
done
```

Is that OK ? (Fix the library versions if you want)